app-web/src/templates/events.js

```diff
@@ -110,6 +110,10 @@ export const EventsPage = ({ data: { allEventbriteEvents, allDevhubTopic } }) =>
             <a href="https://github.com/bcgov/devhub-app-web/issues/new"> here</a> (requires a
             Github account).
           </p>
+          <p>
+            Click here if you interest in any
+            <a href="/past-events"> past events</a>.
```

Comment:
Use `<Link>` component that leverages native gatsby utilities for routing

Reply:
Can I use <ChevronLink> instead?